src/Utils/SchemaPrinter.php

```diff
@@ -396,7 +396,7 @@ private static function printDeprecated($fieldOrEnumVal) : string
         if (empty($reason)) {
```

Comment:
This can be changed to 

```suggestion
        if ($reason === '' || $reason === null) {
```